lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -492,6 +492,17 @@ function _dosomething_reportback_get_permalink_copy_vars($user) {
  * Alters the signup form that is attached to permalink pages.
  */
 function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
+
+  // Make sure a default run is added to the filter
+  if ($form['#id'] == 'views-exposed-form-reportback-files-page-1') {
```

Comment:
So I investigated this a little and found out that actual form to alter is `views_exposed_form`.
Then you need to check
```php
$view = $form_state['view'];
if ($view->name == 'my_view_machine_name' && $view->current_display == 'display_name') {
  // alter your exposed form here
}
```

`$view->name` is your view
`$view->current_display` is this mysterious page 1. You can omit this check.